<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-OU80H79" failures="1" tests="1" name="Default test(failed)" time="0.218" errors="0" timestamp="2023-10-07T10:55:52 IST">
  <testcase classname="com.app.test.Fail_Test_Capture" name="test1" time="0.065">
    <failure type="java.lang.AssertionError" message="expected [Aiit] but found [Aiite]">
      <![CDATA[java.lang.AssertionError: expected [Aiit] but found [Aiite]
at com.app.test.Fail_Test_Capture.test1(Fail_Test_Capture.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="com.app.test.Fail_Test_Capture" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
</testsuite> <!-- Default test(failed) -->
